db/migrate/20200211225056_modify_offers_table.rb

```diff
@@ -0,0 +1,87 @@
+class ModifyOffersTable < ActiveRecord::Migration[5.2]
+  def up
+    change_table :offers, bulk: true do |t|
+      t.rename(:photography_cents, :photography_info)
+      t.rename(:shipping_cents, :shipping_info)
+      t.rename(:insurance_cents, :insurance_info)
+      t.rename(:other_fees_cents, :other_fees_info)
+
+      t.change(
+        :photography_info,
+        :string,
+        using:
+          'CASE WHEN (photography_info IS NOT NULL) ' \
+            "THEN format('%s %s', TRIM(to_char(photography_info / 100, '99999')), currency) END"
```

Comment:
👍 Curious if doing this in SQL is indeed the ideal way, or if a view helper would be a better place? I guess in this case it does make all the data consistent, but also (seems) a little less flexible. (That said, I know very little in terms of idioms for data transforms like this.)

Reply:
Yeah I mean what I expected to see what Ruby, but damn if you know how to do this in SQL then I'm super impressed!! 🏆 

The only lingering thought I have is how we can be sure this works correctly outside of running it on staging and doing some QA on the results. I mean I would write a test for this in the abstract but there really aren't good options for that type of thing when it comes to Rails migrations. I think the pragmatic thing to do is just try it on staging and then be sure we have DB backups went it comes time to promote to production in case something wacky happens.

Thoughts?

Reply:
@damassi Re: flexibility, I'm not sure what you mean. It's a one-time conversion, so I wouldn't anticipate needing any flexibility to make changes. I suspect you might mean something different though.

Reply:
@jonallured Re: expecting Ruby - that makes sense; I have a pretty deep (but secret 🤫) love for SQL, so it's the tool I'd reach for in this kind of situation. This is also the approach I most quickly discovered when researching how to make changes in a migration that are based on existing data. I'm happy to revisit this if you think raw SQL is a bad idea.

Reply:
I do not think this is a bad idea. In fact doing it in Ruby has some tricky edges too. Like what if we rename these things in a future version of the code base? Your SQL version will always work because it's based on the schema while code based off an assumption of the Modeling could be more fragile. Tradeoffs, amiright?

Reply:
And Re: verification - I took a look at the data that's in staging & prod when I wrote this migration. There isn't _a lot_ of data in these fields, but there is _some_ data, which is why I felt it important to make sure we preserve it. 

I verified that the queries I was running inside the `using:` clause returned the expected value for all current data. I also tried to simulate all the possible staging/prod data in my local environment (see "Data snapshot before the migration" and "Data snapshot after the migration" in the PR body), and ran the migration probably a dozen or so times before feeling confident that it was converting things properly.

I definitely agree we should keep an eye on what happens in staging, but I also feel pretty confident that my manual tests covered things.